packages/xstate-store/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@xstate/store",
+  "version": "0.0.1",
+  "description": "Simple stores",
+  "keywords": [
+    "store",
+    "state"
+  ],
+  "author": "David Khourshid <davidkpiano@gmail.com>",
+  "homepage": "https://github.com/statelyai/xstate/tree/main/packages/xstate-store#readme",
+  "license": "MIT",
+  "main": "dist/xstate-store.cjs.js",
+  "module": "dist/xstate-store.esm.js",
+  "exports": {
+    ".": {
+      "types": {
+        "import": "./dist/xstate-store.cjs.mjs",
+        "default": "./dist/xstate-store.cjs.js"
+      },
+      "module": "./dist/xstate-store.esm.js",
+      "import": "./dist/xstate-store.cjs.mjs",
+      "default": "./dist/xstate-store.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/statelyai/xstate.git"
+  },
+  "scripts": {},
+  "bugs": {
+    "url": "https://github.com/statelyai/xstate/issues"
+  },
+  "dependencies": {
+    "@testing-library/react": "^14.2.1",
+    "use-sync-external-store": "^1.2.0"
+  },
+  "devDependencies": {
+    "immer": "^10.0.2",
+    "react": "^18.0.0",
+    "xstate": "^5.8.2"
+  },
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "xstate": "^5.8.2"
```

Comment:
I guess i find it a little bit strange that this gateway lib *does* depend on `xstate`. I know that it's just an optional peer dependency but for many TS users this is really like a regular peer dependency here. Their projects won't compile without `xstate`.

Reply:
that's not what i've tried to convey 😅 i think it would make sense here to just copy-paste those types that you need here and remove a direct dependency (or peer dependency) on XSTate